docs/content/dns/zz_gen_ionos.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Ionos"
+date: 2019-03-03T16:39:46+01:00
+draft: false
+slug: ionos
+---
+
+<!-- THIS DOCUMENTATION IS AUTO-GENERATED. PLEASE DO NOT EDIT. -->
+<!-- providers/dns/ionos/ionos.toml -->
+<!-- THIS DOCUMENTATION IS AUTO-GENERATED. PLEASE DO NOT EDIT. -->
+
+Since: v4.2.0
+
+Configuration for [Ionos](https://ionos.com).
+
+
+<!--more-->
+
+- Code: `ionos`
+
+Here is an example bash command using the Ionos provider:
+
+```bash
+IONOS_API_KEY=xxxxxxxx \
+lego --email myemail@example.com --dns ionos --domains my.example.org run
+```
+
+
+
+
+## Credentials
+
+| Environment Variable Name | Description |
+|-----------------------|-------------|
+| `IONOS_API_KEY` | API Key |
```

Comment:
`API KEY (<prefix>.<secret>)` and maybe a link to https://developer.hosting.ionos.de/docs/getstarted ?